R/extract.R

```diff
@@ -29,6 +29,26 @@
 #' @param ... Not currently used.
 #' @details
 #' These functions supersede the `pull_*()` functions.
+#'
+#' Extracting the underlying fit objects can be helpful for describing the
+#'  model via `print()`, `summarize()`, `plot()`, and so on.
```

Comment:
Would `summary()` be better here, rather than `summarize()`? Also I might suggest mentioning variable importance here, as I think that's one of the big reasons we have folks pull out the engine fit.